### PR TITLE
fix comm typo, add some hu locale strings

### PR DESCRIPTION
**Pull request by ovari**

### CODE REVIEW

File: io.github.endless_sky.endless_sky.desktop [hunk 1]
```diff
@@ -1,9 +1,11 @@
 [Desktop Entry]
 Version=1.5
 Name=Endless Sky
+Name[hu]=Végtelen égbolt
```
Comment (TomGoodIdea) [inline io.github.endless_sky.endless_sky.desktop:4]: I don't know Hungarian, but I think the game name should stay the same in every language; the other languages we have here change only the description.

Comment (ovari) [reply io.github.endless_sky.endless_sky.desktop]: There is no problem with program names being translated; however, updated so can you please merge. Thank you

**Comment by tibetiroka** [comment]
Looks fine. It might be better as `Űrkutatási és harci játék` (notice the extra `i`), but it doesn't matter much.

**Comment by ovari** [comment]
> Looks fine. It might be better as `Űrkutatási és harci játék` (notice the extra `i`), but it doesn't matter much.

Done. Can you please merge? Thank you

**Review by OcelotWalrus — APPROVED** [review]

**Review by tibetiroka — APPROVED** [review]

**Comment by bene-dictator** [comment]
We will need to wait until a maintainer is able to look at this PR; potentially tomorrow.

**Pull request merged**